### PR TITLE
Gidk 1135 ledger id new

### PR DESCRIPTION
**Pull request by shalinee-walkover**
* **What kind of change does this PR introduce?** (Bug fix, feature, docs update, ...)

* **What is the current behavior?** (You can also link to an open issue here)

* **What is the new behavior (if this is a feature change)?**

* **Other information**:

### CODE REVIEW

File: apps/web-giddh/src/app/ledger/components/newLedgerEntryPanel/newLedgerEntryPanel.component.html [hunk 1]
```diff
@@ -269,7 +269,7 @@
                             <td class="select2-parent pdR1">
                                 <div class="form-group">
                                     <label class="default_clr">Voucher Type</label>
-                                    <sh-select #sh [options]="voucherTypeList | async" [isFilterEnabled]="true"
+                                    <sh-select id="ledger-vouchar-type" #sh [options]="voucherTypeList | async" [isFilterEnabled]="true"
```
Comment (ravinderwalkover) [inline apps/web-giddh/src/app/ledger/components/newLedgerEntryPanel/newLedgerEntryPanel.component.html:272]: correct voucher spelling

File: apps/web-giddh/src/app/ledger/components/newLedgerEntryPanel/newLedgerEntryPanel.component.html [hunk 2]
```diff
@@ -316,7 +316,7 @@
                                 <div class="form-group">
                                     <label class="default_clr">Voucher No.</label>
 
-                                    <sh-select [options]="invoiceList" #invoice placeholder="Select Invoice"
+                                    <sh-select id="ledger-vouchar-number" [options]="invoiceList" #invoice placeholder="Select Invoice"
```
Comment (ravinderwalkover) [inline apps/web-giddh/src/app/ledger/components/newLedgerEntryPanel/newLedgerEntryPanel.component.html:319]: correct voucher spelling

File: apps/web-giddh/src/app/ledger/components/updateLedgerEntryPanel/updateLedgerEntryPanel.component.html [hunk 3]
```diff
@@ -758,21 +758,21 @@ <h3 *ngIf="!openDropDown" (dblclick)="openHeaderDropDown()" class="d-inline-bloc
                                         <label>Adjustment Vouchers
                                             <span class="text-danger">*</span>
                                         </label>
-                                        <input type="text" class="form-control mb-1" [(ngModel)]="item.voucherNumber" disabled>
+                                        <input id="update-ledger-adjust-vouchar" type="text" class="form-control mb-1" [(ngModel)]="item.voucherNumber" disabled>
```
Comment (ravinderwalkover) [inline apps/web-giddh/src/app/ledger/components/updateLedgerEntryPanel/updateLedgerEntryPanel.component.html:761]: update voucher spelling

File: apps/web-giddh/src/app/ledger/components/updateLedgerEntryPanel/updateLedgerEntryPanel.component.html [hunk 4]
```diff
@@ -758,21 +758,21 @@ <h3 *ngIf="!openDropDown" (dblclick)="openHeaderDropDown()" class="d-inline-bloc
...
+                                        <input id="update-ledger-inclusive-tax" type="text" class="form-control mb-1" decimalDigitsDirective [mask]="vm.inputMaskFormat" [prefix]="vm.selectedPrefixForCurrency" (blur)="adjustedInvoiceAmountChange()" *ngIf="selectedAdvanceReceiptAdjustInvoiceEditMode" [(ngModel)]="item.adjustmentAmount.amountForAccount">
                                     </div>
                                 </div>
                                 <div *ngIf="item.voucherType === 'receipt' && item.subVoucher === 'ADVANCE_RECEIPT'" class="col-sm-2 pl-05">
                                     <div class="form-group">
                                         <label>Tax</label>
-                                        <input type="text" class="form-control mb-1" decimalDigitsDirective [mask]="vm.inputMaskFormat" [prefix]="vm.selectedPrefixForCurrency" [(ngModel)]="item.calculatedTaxAmount" disabled>
+                                        <input type="text" id="update-ledger-vouchar-tax" class="form-control mb-1" decimalDigitsDirective [mask]="vm.inputMaskFormat" [prefix]="vm.selectedPrefixForCurrency" [(ngModel)]="item.calculatedTaxAmount" disabled>
```
Comment (ravinderwalkover) [inline apps/web-giddh/src/app/ledger/components/updateLedgerEntryPanel/updateLedgerEntryPanel.component.html:775]: update voucher spelling

File: apps/web-giddh/src/app/ledger/components/updateLedgerEntryPanel/updateLedgerEntryPanel.component.html [hunk 5]
```diff
@@ -867,7 +867,7 @@ <h3 *ngIf="!openDropDown" (dblclick)="openHeaderDropDown()" class="d-inline-bloc
                                 <td class="select2-parent pdR1">
                                     <div class="form-group">
                                         <label class="default_clr">Voucher Type</label>
-                                        <sh-select [options]="vm.voucherTypeList" [isFilterEnabled]="true"
+                                        <sh-select id="update-ledger-vouchar-type" [options]="vm.voucherTypeList" [isFilterEnabled]="true"
```
Comment (ravinderwalkover) [inline apps/web-giddh/src/app/ledger/components/updateLedgerEntryPanel/updateLedgerEntryPanel.component.html:870]: update voucher spelling

File: apps/web-giddh/src/app/ledger/components/updateLedgerEntryPanel/updateLedgerEntryPanel.component.html [hunk 6]
```diff
@@ -941,7 +941,7 @@ <h3 *ngIf="!openDropDown" (dblclick)="openHeaderDropDown()" class="d-inline-bloc
                                 <td class="pdR1">
                                     <div class="form-group">
                                         <label class="default_clr">Voucher No.</label>
-                                        <input type="text" placeholder="Voucher no." class="form-control"
+                                        <input id="update-ledger-vouchar-number" type="text" placeholder="Voucher no." class="form-control"
```
Comment (ravinderwalkover) [inline apps/web-giddh/src/app/ledger/components/updateLedgerEntryPanel/updateLedgerEntryPanel.component.html:944]: update voucher spelling

**Review by ravinderwalkover — CHANGES_REQUESTED** [review]

File: apps/web-giddh/src/app/ledger/components/newLedgerEntryPanel/newLedgerEntryPanel.component.html (hunk 2 above)
Comment (shalinee-walkover) [reply apps/web-giddh/src/app/ledger/components/newLedgerEntryPanel/newLedgerEntryPanel.component.html]: Done, updated all voucher spell

**Review by rohit-walkover — APPROVED** [review]

File: apps/web-giddh/src/app/ledger/components/updateLedgerEntryPanel/updateLedgerEntryPanel.component.html (hunk 6 above)
Comment (shalinee-walkover) [reply apps/web-giddh/src/app/ledger/components/updateLedgerEntryPanel/updateLedgerEntryPanel.component.html]: All spell corrections done

**Review by ravinderwalkover — APPROVED** [review]

**Review by prateek-giddh — APPROVED** [review]

**Pull request merged**